src/docfx/build/xref/XrefResolver.cs

```diff
@@ -130,35 +130,32 @@ internal class XrefResolver
             return (error, xrefSpec, href);
         }
 
-        public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
+        public XrefMapModel ToXrefMapModel()
         {
             var repositoryBranch = _repository?.Branch;
             var basePath = _config.BasePath.ValueWithLeadingSlash;
 
             var references = Array.Empty<ExternalXrefSpec>();
             var externalXrefs = Array.Empty<ExternalXref>();
 
-            if (!isLocalizedBuild)
-            {
-                references = _internalXrefMap.Value.Values
-                    .Select(xrefs =>
-                    {
-                        var xref = xrefs.First();
+            references = _internalXrefMap.Value.Values
+                .Select(xrefs =>
+                {
+                    var xref = xrefs.First();
 
-                        // DHS appends branch information from cookie cache to URL, which is wrong for UID resolved URL
-                        // output xref map with URL appending "?branch=master" for master branch
-                        var query = _config.UrlType == UrlType.Docs && repositoryBranch != "live"
-                            ? $"?branch={repositoryBranch}" : "";
+                    // DHS appends branch information from cookie cache to URL, which is wrong for UID resolved URL
+                    // output xref map with URL appending "?branch=master" for master branch
+                    var query = _config.UrlType == UrlType.Docs && repositoryBranch != "live"
+                    ? $"?branch={repositoryBranch}" : "";
 
-                        var href = UrlUtility.MergeUrl($"https://{_xrefHostName}{xref.Href}", query);
+                    var href = UrlUtility.MergeUrl($"https://{_xrefHostName}{xref.Href}", query);
 
-                        return xref.ToExternalXrefSpec(href);
-                    })
-                    .OrderBy(xref => xref.Uid)
-                    .ToArray();
+                    return xref.ToExternalXrefSpec(href);
+                })
+                .OrderBy(xref => xref.Uid)
+                .ToArray();
 
-                externalXrefs = _jsonSchemaTransformer().GetValidateExternalXrefs();
-            }
+            externalXrefs = _jsonSchemaTransformer().GetValidateExternalXrefs();
 
             var model =
                 new XrefMapModel { References = references, ExternalXrefs = externalXrefs, RepositoryUrl = _repository?.Url, DocsetName = _config.Name.Value };
```

Comment:
Is there a way to identify the locale of an xrefmap?

Reply:
We should be able to have the info of localization during build.
But what is it used for?

Reply:
It's not useful for now, we can add later if needed.